evap/static/scss/components/_grid.scss

```diff
@@ -122,7 +132,30 @@
     }
 }
 
-@each $col in name, semester, responsible, participants, result {
+.textanswer-review-grid {
+    @extend %grid-row;
+
+    gap: 0;
+    grid:
+        "answer edit review flag"
+        / auto  min-content min-content min-content;
+
+    min-height: 0;
+    padding: 0.5rem;
+
+    .grid-row > div {
+        height: 100%;
+        padding: 0.5rem;
+
+        &:not(:first-child) {
+            display: flex;
+            align-items: center;
+            padding-left: 0.75rem;
+        }
```

Comment:
The HTML has this information duplicated as the `div` that represent the cells of the table inside a `div.grid-row` have a class of either `justify-content-left` or `justify-content-center`. I do not strongly prefer either place for having this information, but if possible, I would not want it duplicated. Do you think we can achieve this?